pkgs/development/python-modules/minc2-simple/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage, fetchFromGitHub, stdenv, cmake, minc2_simple,
+  cffi, six, numpy, scipy, pytest, minc_tools, mni_autoreg }:
+
+buildPythonPackage rec {
+  pname    = "minc2-simple";
+  name     = "${pname}-2018-12-20";
+
+  src = fetchFromGitHub {
+    owner  = "vfonov";
+    repo   = pname;
+    rev    = "e04414eb74052d8a6f9fb4bedda7f0894e86ac82";
+    sha256 = "1miy942ibbcikg0k45d7342ags2gpr29rvhvspwxqgv2jnpcwg05";
+  };
+
+  checkInputs = [ pytest minc_tools mni_autoreg ];
+  propagatedBuildInputs = [ cffi six numpy scipy minc2_simple ];
+
+  preConfigure = "cd python";
+
+  testPhase = "pytest test";
```

Comment:
testPhase does not exist, it's `checkPhase`

Reply:
Removed.